helm/nessie/templates/ingress.yaml

```diff
@@ -35,8 +36,9 @@ spec:
         paths:
           {{- range .paths }}
           - path: {{ . }}
+            pathType: ImplementationSpecific
             backend:
-              {{- if semverCompare ">=1.22-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
alternatively we could also prefix this with `$` (`$.Capabilities.KubeVersion.GitVersion`) but the less repeatable code the better :)